Products/Zuul/catalog/model_catalog_tool.py

```diff
@@ -189,15 +189,15 @@ def _build_query(self, types=(), paths=(), depth=None, query=None, filterPermiss
         search_query = And(*partial_queries)
         return (search_query, not_indexed_user_filters)
 
-    def search_model_catalog(self, query, start=0, limit=None, order_by=None, reverse=False,
+    def search_model_catalog(self, query, start=0, cursor="*", limit=None, order_by=None, reverse=False,
```

Comment:
Can you use the constant name from modelindex rather than hard-coding the initial cursor mark?